homeassistant/components/elevenlabs/tts.py

```diff
@@ -118,7 +119,7 @@ def __init__(
 
         # Entity attributes
         self._attr_unique_id = entry_id
-        self._attr_name = title
+        self._attr_name = f"{title} {model.name}"
```

Comment:
So in theory the name should be `None` here to match the entry title. I know we looked at this before but still haven't found out why this is the case

Reply:
Yeah, I think it's a bit odd behaviour. I could go for None, though I am not sure I like it.

Reply:
I don't see the problem?
The name would be `ElevenLabs` unless the user renames it in that case.

Reply:
None is impossible due to this check: https://github.com/home-assistant/core/blob/ad89004189195654c7c4e576fe314adc636888a9/homeassistant/components/tts/__init__.py#L778

Reply:
Pinging @synesthesiam here because I think this check is too strict - in our case, None should be a valid name